### ISSUE TITLE
when train 5-layer ANN, it hard to overfit in a small dataset.

### ISSUE THREAD

**Issue by smart-donkey** [smart-donkey/cs231n#1]
Maybe it's because weight is not batch normalized in hidden layer?
